lib/posix-environ/environ.c

```diff
@@ -7,6 +7,28 @@
 #include <uk/essentials.h>
 #include "environ.h"
 
+#define DECLARE_LIBPOSIX_ENVIRON_ENV_VAR(x)			\
+	static char _e##x[] = CONFIG_LIBPOSIX_ENVIRON_ENVP##x
+
+#define LIBPOSIX_ENVIRON_ENV_VAR(x)		_e##x
```

Comment:
If you swap `LIBPOSIX_ENVIRON_ENV_VAR(x)` with `DECLARE_LIBPOSIX_ENVIRON_ENV_VAR(x)` you can even use `LIBPOSIX_ENVIRON_ENV_VAR(x)` within `DECLARE_LIBPOSIX_ENVIRON_ENV_VAR(x)` :)